### PR TITLE
libsais 2.7.3

### PR DESCRIPTION
**Pull request by p-linnane**
Created by `brew bump`

---

Created with `brew bump-formula-pr`.<details>
  <summary>release notes</summary>
  <pre>CMake script for library build and integration with other projects.</pre>
</details>

### CODE REVIEW

File: Formula/libsais.rb [hunk 1]
```diff
@@ -16,13 +16,15 @@ class Libsais < Formula
...
 
-    include.install Dir.glob("*.h")
-    lib.install shared_library("libsais"), "libsais.a"
+  def install
+    system "cmake", "-S", ".", "-B", "build",
+                    *std_cmake_args
+    system "cmake", "--build", "build"
+    lib.install "build/liblibsais.a"
+    lib.install_symlink lib/"liblibsais.a" => "libsais.a"
+    include.install "include/libsais.h"
```
Comment (carlocab) [inline Formula/libsais.rb:27]: No `cmake --install build`?

File: Formula/libsais.rb [hunk 2]
```diff
@@ -16,13 +16,15 @@ class Libsais < Formula
...
-    system ENV.cc, "-c", *Dir.glob("*.c")
-    system ENV.cc, "-shared", "-o", shared_library("libsais"), *Dir.glob("*.o")
-    system "ar", "rcs", "libsais.a", *Dir.glob("*.o")
+  depends_on "cmake" => :build
 
-    include.install Dir.glob("*.h")
-    lib.install shared_library("libsais"), "libsais.a"
+  def install
+    system "cmake", "-S", ".", "-B", "build",
+                    *std_cmake_args
```
Comment (carlocab) [inline Formula/libsais.rb:23]:
```suggestion
    system "cmake", "-S", ".", "-B", "build",
                    "-DBUILD_SHARED_LIBS=ON",
                    *std_cmake_args
```
You'll need to adjust the stuff installed below tho.

File: Formula/libsais.rb (hunk 1 above)
Comment (p-linnane) [reply Formula/libsais.rb:27]:
```
==> cmake --install build
-- Install configuration: "Release"
Error: Empty installation
```

File: Formula/libsais.rb (hunk 2 above)
Comment (p-linnane) [reply Formula/libsais.rb]: Looks like they use `LIBSAIS_BUILD_SHARED_LIB` here.

**Review by chenrui333 — DISMISSED** [review]

**Pull request merged**